src/bootstrap/bootstrap.py

```diff
@@ -732,19 +732,26 @@ def build_bootstrap(self, color):
             (os.pathsep + env["LIBRARY_PATH"]) \
             if "LIBRARY_PATH" in env else ""
 
+        # Export Stage0 snapshot compiler related env variables
+        build_section = "target.{}".format(self.build)
+        host_triple_sanitized = self.build.replace("-", "_")
+        var_data = {
+            "CC": "cc", "CXX": "cxx", "LD": "linker", "AR": "ar", "RANLIB": "ranlib"
+        }
+        for var_name, toml_key in var_data.items():
+            toml_val = self.get_toml(toml_key, build_section)
+            if toml_val != None:
+                env[f"{var_name}_{host_triple_sanitized}"] = toml_val
```

Comment:
Looks like this is still using f-strings.

Reply:
Done